src/index.js

```diff
@@ -41,117 +42,125 @@ const App = () => {
-                  },
-                ]}
-              />
-
-              <Main>
-
-                <Switch>
-
-                  <Route 
-                    exact
-                    path='/'
-                    render={() => (
-                      <Request url={data.instructor_url}>
-                        {({data}) => (
-                          <div>
-                            <Overview instructor={data} />
-                          </div>
-                        )}
-                      </Request>
-                    )}
-                  />
-
-                  <Route 
-                    exact
-                    path='/lessons/new'
-                    render={() => (
-                      <Request url={data.instructor_url}>
-                        {({data}) => (
-                          <New instructor={data} />
-                        )}
-                      </Request>
-                    )}
-                  />
-
-                  <Route 
-                    path={`/lessons/:slug`}
-                    render={({match}) => (
-                      <Request url={`/api/v1/lessons/${match.params.slug}`}>
-                        {({data}) => (
-                          <Lesson lesson={data} />
-                        )}
-                      </Request>
-                    )}
-                  />
-
-                  <Route 
-                    path={`/instructors/:slug`}
-                    render={({match}) => (
-                      <Request url={`/api/v1/instructors/${match.params.slug}`}>
-                        {({data}) => (
-                          <Overview instructor={data} />
-                        )}
-                      </Request>
-                    )}
-                  />
-
-                  <Route
-                    path='/instructors'
-                    render={() => (
-                      <Request url='/api/v1/instructors'>
-                        {({data}) => (
-                          <Instructors instructors={data} />
-                        )}
-                      </Request>
-                    )}
-                  />
-
-                  <Route component={RouteNotFound} />
-
-                </Switch>
-
-              </Main>
-
-            </div>
+            <Request url={data.instructor_url}>
+              {({data}) => {
+
+                const instructor = data
+
+                return (
+                  <div>
+
+                    <Navigation
+                      items={[
+                        {
+                          text: overviewTitleText,
+                          action: '/',
+                        },
+                        {
+                          text: newLessonsTitleText,
+                          action: '/lessons/new',
+                        },
+                        {
+                          text: queueTitleText,
+                          action: temporaryQueueUrl,
+                        },
```

Comment:
Adds a temporary "Queue" link to navigation that links to egghead.io, until we build a queue in the Instructor Center directly